bubble_sort/Cpp/bubble_sort.cpp

```diff
@@ -1,4 +1,5 @@
-#include <stdio.h>
+#include <iostream>
+using namespace std;
 
 void swap(int *xp, int *yp) {
 	int temp = *xp;
```

Comment:
Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*